### DISCUSSION TITLE
Android 15 Beta

### DISCUSSION THREAD

**Discussion by bikemike** [kyujin-cho/pixel-volte-patch#290] [General]
Has anyone tried this with the Android 15 Beta(AP31.240322.018)? I would like to try the beta but want to make sure I can still enable backup calling though Pixel IMS after updating.

**Comment by msilcher** [comment]
good question! Android 15 is around the corner, did anyone test it so far?

**Comment by exolon** [comment]
Works fine on my P7P...

**Comment by bikemike** [comment]
Yes, I used it about a mouth ago after updating to Android 15 too and it worked.

**Reply by dperez83** [reply]
I am having problems after updating my Pixel 6 to Android 15. IMS registration + VoLTE + VoWifi worked correctly on Android 14 but after the update to 15 IMS registration never succeeds (so no VoLTE and no VoWifi).

I already tried all troubleshooting steps from https://github.com/kyujin-cho/pixel-volte-patch/blob/main/docs/troubleshooting.en.md#volte-does-not-work-after-various-situations with no luck.
